python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -313,6 +313,11 @@ class ImageDifferenceConfig(pipeBase.PipelineTaskConfig,
         dtype=float, default=0.5,
         doc="Match radius (in arcseconds) for DiaSource to Source association"
     )
+    requiredTemplateFraction = pexConfig.Field(
+        dtype=float, default=0.12,
+        doc="Do not attempt to run task if template covers less than this fraction of pixels."
+        "Setting to 0 will always attempt image subtraction"
```

Comment:
17% is also on the table as the minimum needed to be able to drop. this line excluding two visits that fail because they have 12 and 16% template coverage.

Reply:
I agree 17% would also be a reasonable default. We tend to get more junk than good data from the edge regions, so it's reasonable to be more aggressive about throwing it out. Please do explain in the docstring where the number is coming from, though.